app/views/welcome/index.html.erb

```diff
@@ -2,7 +2,28 @@
     <%= "Search Performance Explorer" %>
 <% end %>
 
+<% content_for :head do %>
+  <script>
+    $(document).ready(function() {
+      $('input[type="radio"]').click(function() {
+        if($(this).attr('id') == 'info_enhanced') {
+          $('.enhanced-checkbox').show();
+        }
+        else {
+          $('.enhanced-checkbox').hide();
+        }
+      });
+    });
+  </script>
+<% end %>
```

Comment:
scripts should ideally go later in the page than content, as such can we move this to the footer